package.json

```diff
@@ -95,29 +92,32 @@
     "@storybook/react": "5.2.5",
     "@types/chart.js": "2.8.10",
     "@types/enzyme": "3.10.3",
+    "@types/highlight.js": "9.12.3",
     "@types/history": "4.7.3",
     "@types/jest": "24.0.21",
     "@types/jquery": "3.3.31",
     "@types/lodash": "4.14.144",
     "@types/mark.js": "8.11.4",
+    "@types/marked": "0.6.5",
     "@types/moment-timezone": "0.5.12",
     "@types/node": "12.12.5",
-    "@types/react": "16.8.24",
+    "@types/react": "^16.9.19",
```

Comment:
Should these all be locked to exact versions?

Reply:
So I think these changes came from fixing the issues identified by `npm audit`. Happy to change them to exact versions though if that's preferred - that's why I've got you frontend experts reviewing this PR :)